common/src/domain/event/event.ts

```diff
@@ -0,0 +1,69 @@
+import { GREPA, LEPPA } from 'src/domain/berry';
+import { AddOneIngredient } from 'src/domain/event/event-util';
+import { Pokemon } from 'src/domain/pokemon';
+
+export interface Event {
+  name: string;
+  startDate: string;
+  endDate: string;
+  statModifier: (mon: Pokemon) => Pokemon;
+}
+
+export const ENTEI_RESEARCH: Event = {
```

Comment:
I guess listing all the events in here will eventually become messy. Do you think we could take a similar approach to how we do pokemon? Perhaps one folder for each year, so right now we would have 2023 and 2024

Reply:
I'm not convinced this is necessary, as we've still got way way fewer events than Pokemon. I'd be inclined to punt this until it becomes more of an issue?

Reply:
That's fine, this isn't a big issue to me